doc/tutorial/tutorial.md

```diff
@@ -734,7 +734,15 @@ function addr_port(ip: ip_addr_t, proto: string, preferred_port: bit<16>): strin
 
 The result computed by a function is the value of the last expression evaluated
 or the value produced by the first `return` statement encountered when
-evaluating the function.  If the `else` is missing the value `()` (empty tuple)
+evaluating the function.  A semicolon at the end of a sequence of expressions
```

Comment:
I am not sure this is a good idea. I think it will trip a lot of people who come from a C-like language.
I think it should be a warning.

Reply:
There is no room for confusion here.  If the expression is expected to return a type that is not an empty tuple, you will get a type error.  If it returns `()`, then discarding it is a no-op.

Reply:
the type error may be confusing, especially if it involves some form of unification.